docs/quickstarts/sso-quickstart.md

```diff
@@ -119,24 +119,24 @@ Complete the following steps to try out an Excel, Word, or PowerPoint add-in.
 
 Complete the following steps to try out an Outlook add-in.
 
-1. When the SSO configuration process completes, run the following command to build the project and start the local web server.
+1. When the SSO configuration process completes, run the following command to build the project and start the local web server, and sideload your add-in in the previously selected Office client application.
 
     > [!NOTE]
     > Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides. You may also have to run your command prompt or terminal as an administrator for the changes to be made.
 
     ```command&nbsp;line
-    npm run dev-server
+    npm start
```

Comment:
This won't launch Outlook and sideload. You still need to link to those instructions.

Reply:
Kept sideloading instructions